boa/contracts/base_evm_contract.py

```diff
@@ -76,9 +82,9 @@ def _handle_child_trace(computation, env, return_trace):
     else:
         child_trace = child_obj.stack_trace(child)
 
-    if child_trace.last_frame.dev_reason and not return_trace.last_frame.dev_reason:
+    if child_trace.dev_reason and not return_trace.dev_reason:
```

Comment:
this change looks spooky. maybe we should check if `last_frame is not None`?

Reply:
ditto

Reply:
we should still test for `is not None`, as you could have an empty dev reason (which is valid)

Reply:
https://github.com/vyperlang/titanoboa/pull/213/commits/b5b1f9e3fa97f80a3278157a8b721019b9a30c8f

Reply:
the return value of `dev_reason` could still be an empty string!

Reply:
https://github.com/vyperlang/titanoboa/pull/213/commits/0ca0ea7459f54ac7d69d57e944a1fc9a65422158